src/styles/main.scss

```diff
@@ -1,3 +1,10 @@
-body {
-  margin: 0;
-}
+@import "./utils/reset.scss";
+@import "./utils/normalize.scss";
+@import "./utils/_variables.scss";
```

Comment:
Consider removing the underscore prefix from '_variables.scss' and '_functions.scss' as they are being imported directly.